pom.xml

```diff
@@ -414,9 +414,9 @@
   </distributionManagement>
 
   <scm>
-    <url>https://github.com/folio-org/${artifactId}</url>
-    <connection>scm:git:git://github.com/folio-org/${artifactId}.git</connection>
-    <developerConnection>scm:git:git@github.com:folio-org/${artifactId}.git</developerConnection>
+    <url>https://github.com/folio-org/${project.artifactId}</url>
```

Comment:
maven was yelling about `${artifactId}` being deprecated, so decided to update this while I was in here